src/test/java/nextstep/cucumber/steps/PathStepDef.java

```diff
@@ -0,0 +1,74 @@
+package nextstep.cucumber.steps;
+
+import static nextstep.fixture.SubwayScenarioFixtureCreator.*;
+import static nextstep.utils.resthelper.ExtractableResponseParser.*;
+import static nextstep.utils.resthelper.PathRequestExecutor.*;
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.http.HttpStatus;
+
+import io.cucumber.datatable.DataTable;
+import io.cucumber.java8.En;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+
+/**
+ * @author : Rene Choi
+ * @since : 2024/02/23
+ */
+public class PathStepDef implements En {
+
+	private ExtractableResponse<Response> response;
+
+	public PathStepDef() {
+		Given("지하철 역들이 생성되어 있다:", (DataTable stationsTable) -> {
+			List<Map<String, String>> stations = stationsTable.asMaps(String.class, String.class);
+			stations.forEach(station -> createStation(station.get("name")));
+		});
+
+		And("지하철 노선들이 생성되어 있다:", (DataTable linesTable) -> {
+			List<Map<String, String>> lines = linesTable.asMaps(String.class, String.class);
+			lines.forEach(line -> {
+				String name = line.get("line");
+				long upStationId = Long.parseLong(line.get("upStationId"));
+				long downStationId = Long.parseLong(line.get("downStationId"));
+				long distance = Long.parseLong(line.get("distance"));
+				createLine(name, upStationId, downStationId, distance);
+			});
+		});
+
+		And("다음 구간이 추가되어 있다:", (DataTable sectionsTable) -> {
+			List<Map<String, Long>> sections = sectionsTable.asMaps(String.class, Long.class);
+			sections.forEach(section -> {
+				Long lineId = section.get("lineId");
+				Long upStationId = section.get("upStationId");
+				Long downStationId = section.get("downStationId");
+				Long distance = section.get("distance");
+
+				createSection(lineId, upStationId, downStationId, distance);
+			});
+		});
+
+		When("교대역에서 양재역까지의 최단 경로를 조회하면", () -> {
+			response = executeFindPathRequest(1L, 3L);
+		});
```

Comment:
최단 경로를 조회하는 부분을 모듈화해보면 어떨까요?
```suggestion
When("{string}에서 {string}까지의 최단 경로를 조회하면", (source, target) -> {
			response = executeFindPathRequest(source, target);
		});
```